pymc/testing.py

```diff
@@ -653,34 +653,34 @@ def check_selfconsistency_discrete_logcdf(
             )
 
 
-def assert_moment_is_expected(model, expected, check_finite_logp=True):
+def assert_support_point_is_expected(model, expected, check_finite_logp=True):
```

Comment:
This is also user facing (in that we use this utility in other libraries), so should also add a wrapper with a deprecation warning.

Reply:
I added option for using old name